cfme/common/provider_views.py

```diff
@@ -115,6 +116,7 @@ class ProviderDetailsSummaryView(View):
         represents Details page when it is switched to Summary aka Tables view
         """
         summary = ParametrizedSummaryTable()
+        smart_management = SummaryTable(title="Smart Management")
```

Comment:
You do not need this, the Smart Management table is defined by the ParametrizedSummaryTable.

Example:
```python
view = navigate_to(provider, 'Details')
fields = view.entities.summary('Smart Management').fields
```

Reply:
@jawatts Thanks, I will try this